src/collective/volto/formsupport/restapi/services/submit_form/field.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass
+from typing import List, Optional, Any
+
+
+@dataclass
+class Field:
+    field_id: str
+    field_type: str
+    id: str
+    label: str
+    required: str
+    show_when_when: str
+    submitted_value: Any
+    input_values: Optional[List[dict]] = None
+    internal_value: Optional[dict] = None
+    widget: Optional[str] = None
+
+    def get_display_value(self):
+        if self.internal_value:
+            return self.internal_value.get(self.submitted_value, self.submitted_value)
+        return self.submitted_value
+
+    @property
+    def send_in_email(self):
+        return True
+
+
+class YesNoField(Field):
+    def get_display_value(self):
+        if self.internal_value:
+            if self.submitted_value is True:
+                return self.internal_value.get("yes")
+            elif self.submitted_value is False:
+                return self.internal_value.get("yes")
```

Comment:
bug here.
and you can write this a lot simpler as ```self.internal_value.get("yes" if self.submitted_value else "no")```
however I why is internal_value keyed by "yes" and "no" rather than boolean?

Any why is internal_value storing display_values? I don't get the naming